infra/resources/prod/italynorth.tf

```diff
@@ -0,0 +1,11 @@
+module "apim_itn" {
+  source = "../_modules/apim"
+
+  env_short = local.env_short
+
+  apim_name                = data.azurerm_api_management.apim_itn.name
+  apim_resource_group_name = data.azurerm_api_management.apim_itn.resource_group_name
+
+  api_host_name       = "api.io.pagopa.it"
```

Comment:
Are we already capable of deploying this api host name?

Reply:
I think so, but I don't think it's very relevant because as soon as the restore is completed this value will be inserted anyway